man/man8/sslsniff.8

```diff
@@ -46,6 +46,8 @@ Show data as hexdump instead of trying to decode it as UTF-8
 \-\-max-buffer-size SIZE
 Sets maximum buffer size of intercepted data. Longer values would be truncated.
 Default value is 8 Kib, maximum possible value is a bit less than 32 Kib.
+\-v, \-\-verbose
+Trace SSL handshake calls and show function latency in ms.
```

Comment:
Typically -v option is used to emit verbose output and it does not change functionality. Here, -v actually changed functionality, e.g., tracing SSL handshake and show function latency. So '-v' is not a good option to use here.

We do have funclatency tool to trace function latency. Do you think that can be used for latency tracing? I understand latency tracing in sslsniff can get more context information (e.g., len). But if you don't care about len, then I think funclatency should be good enough for you, and this patch is really not necessary. So please explain the use case why funclatency is not sufficient here.

If we do need to trace latency in sslsniff tool, I suggest we have two options: "-l" option enables tracing latency and "--handshake" tracing handshake functions. And "--handshake" is only enabled if "-l" is present. Please make it clear in the man page and "-h" output.

Reply:
Thanks for the feedback. I'm not sure if it's fine to change original output format, so I chose verbose option to add new functionality.

SSL handshake latency is useful when shown together with SSL_read or SSL_write. So it's more like using trace tool with function latency. funclatency has latency histograms, good for summary, and I also add similar tools for crypto functions in bpftrace [PR 2117](https://github.com/iovisor/bpftrace/pull/2117). Here I'm trying to help sslsniff tool to show the overall SSL handshake overhead before secure connection is ready for SSL_read or SSL_write. 

I‘ll use "-l" and "--handshake" options to replace the verbose option as per suggestion.

Reply:
> SSL handshake latency is useful when shown together with SSL_read or SSL_write. So it's more like using trace tool with function latency. funclatency has latency histograms, good for summary, and I also add similar tools for crypto functions in bpftrace [PR 2117](https://github.com/iovisor/bpftrace/pull/2117). Here I'm trying to help sslsniff tool to show the overall SSL handshake overhead before secure connection is ready for SSL_read or SSL_write.

Thanks for explanation. This is important and please include such description in the commit message.